.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: node_js
+node_js:
+  - stable
+  - 8.9.4
+
+
+
+cache:
+directories:
+  - "node_modules"
+
+
+before_script:
+
+  - psql -c 'create database travis;' -U postgres
+
+install:
+  - npm install
+script:
+  - npm run test
+  - npm run coveralls
+
```

Comment:
Is there a script in the package.json that handles the command on line 21?

Reply:
Not